### PR TITLE
Dont render empty stock descriptions

### LINKED ISSUE #13415: WC 2.7 RC1 | Stock paragraph gets output in single product even if stock management is disabled
## EXPLANATION OF THE ISSUE

On WooCommerce 2.7.0-RC1 the contents of the stock.php template are echoed in the single product even if stock management is disabled either globally or per product.

## STEPS TO REPRODUCE THE ISSUE

1. Create and publish new product without enabling stock management.
2. Inspect the new product and locate an empty `<p class="stock in-stock"></p>` above the cart form.

## SYSTEM STATUS REPORT

```
### WordPress Environment ###

Home URL: http://192.168.0.106
Site URL: http://192.168.0.106
WC Version: 2.7.0
Log Directory Writable: ✔
WP Version: 4.7.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: 7.0.10
PHP Post Max Size: 64 MB
PHP Time Limit: 300
PHP Max Input Vars: 1000
cURL Version: 7.45.0
OpenSSL/1.0.2h

SUHOSIN Installed: –
MySQL Version: 5.7.14
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.7.0
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ❌ The MaxMind GeoIP Database does not exist - Geolocation will not function. You can download and install it manually from http://dev.maxmind.com/geoip/legacy/geolite/ to the path: . Scroll down to "Downloads" and download the "Binary / gzip" file next to "GeoLite Country". Please remember to uncompress GeoIP.dat.gz and upload the GeoIP.dat file only.

### Security ###

Secure connection (HTTPS): ❌Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (3) ###

WooCommerce Beta Tester: by Mike Jolley – 1.0.1
WooCommerce: by Automattic – 2.7.0-RC1
WordPress Importer: by wordpressdotorg – 0.6.3

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)


### WC Pages ###

Shop base: #9 - /shop/
Cart: #10 - /cart/
Checkout: #11 - /checkout/
My account: #12 - /my-account/

### Theme ###

Name: Twenty Sixteen
Version: 1.3
Author URL: https://wordpress.org/
Child Theme: ❌ – If you're modifying WooCommerce on a parent theme you didn't build personally
then we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –
```

### PR DESCRIPTION
**Pull request by claudiulodro**
Fixes #13415.

Don't try and render the stock template if there is no content to go inside it. I've made sure to keep all the filters working correctly.

### CODE REVIEW

File: includes/wc-template-functions.php [hunk 1]
```diff
@@ -2500,17 +2500,21 @@ function wc_display_product_attributes( $product ) {
...
 
+	$html = "";
 	$availability = $product->get_availability();
 
-	wc_get_template( 'single-product/stock.php', array(
-		'product'      => $product,
-		'class'        => $availability['class'],
-		'availability' => $availability['availability'],
-	) );
+	if ( "" !== $availability['availability'] ) {
```
Comment (mikejolley) [inline includes/wc-template-functions.php]: Use single quotes here.

Comment (mikejolley) [inline includes/wc-template-functions.php]: Or an `empty`

**Review by mikejolley — CHANGES_REQUESTED** [review]
Just needs a small change then you can merge :)

**Review by claudiosanches — APPROVED** [review]
Looks good now.

**Pull request merged**